core/dispatcher/src/whisk/core/container/ContainerUtils.scala

```diff
@@ -170,18 +172,19 @@ trait ContainerUtils extends Logging {
      * Synchronously runs the given docker command returning stdout if successful.
      */
     def runDockerCmd(skipLogError: Boolean, args: Seq[String])(implicit transid: TransactionId): DockerOutput = {
-        getDockerCmd(dockerhost) map { _ ++ args } map { info(this, s"runDockerCmd: transid = $transid"); SimpleExec.syncRunCmd(_)(transid) } match {
+        getDockerCmd(dockerhost) map { _ ++ args } map { info(this, s"runDockerCmd: transid = $transid", LogMarkerToken("invoker", s"docker.${args(0)}", "start")); SimpleExec.syncRunCmd(_)(transid) } match {
             case Some((stdout, stderr, exitCode)) =>
                 if (exitCode == 0) {
+                    info(this, "", LogMarkerToken("invoker", s"docker.${args(0)}", "finish"))
                     Some(stdout.trim)
```

Comment:
Args is a part of the marker, because args(0) is the docker command, that will be executed. E.g run (for docker run ...)